crates/router/src/configs/defaults.rs

```diff
@@ -437,48 +413,302 @@ impl Default for super::settings::RequiredFields {
+                                    },
+                                    RequiredFieldInfo {
+                                        required_field: "billing.address.first_name".to_string(),
+                                        display_name: "first_name".to_string(),
+                                        field_type: enums::FieldType::UserBillingName,
+                                    }
+                                ],
+                            ),
+                            (
+                                enums::Connector::Globalpay,
+                                vec![RequiredFieldInfo {
+                                    required_field: "billing.address.country".to_string(),
+                                    display_name: "country".to_string(),
+                                    field_type: enums::FieldType::UserAddressCountry {
+                                        options: vec!["US".to_string(), "IN".to_string()],
+                                    },
+                                }],
+                            ),
+                            (
+                                enums::Connector::Iatapay,
                                 vec![RequiredFieldInfo {
-                                    required_field: "description".to_string(),
-                                    display_name: "description".to_string(),
-                                    field_type: enums::FieldType::Text,
+                                    required_field: "billing.address.country".to_string(),
+                                    display_name: "country".to_string(),
+                                    field_type: enums::FieldType::UserAddressCountry {
+                                        options: vec!["AT".to_string(), "BE".to_string(), "DE".to_string(), "ES".to_string(), "FI".to_string(), "GB".to_string(), "IN".to_string(), "IT".to_string(), "NL".to_string(), "PT".to_string() ],
+                                    },
                                 }],
                             ),
                             (
-                                enums::Connector::Payu,
+                                enums::Connector::Multisafepay,
+                                vec![
+                                    RequiredFieldInfo {
+                                        required_field: "billing.address.first_name".to_string(),
+                                        display_name: "first_name".to_string(),
+                                        field_type: enums::FieldType::UserBillingName,
+                                    },
+                                    RequiredFieldInfo {
+                                        required_field: "billing.address.line1".to_string(),
+                                        display_name: "line1".to_string(),
+                                        field_type: enums::FieldType::UserAddressline1,
```

Comment:
We might have to consider renaming this and `UserAddressline2` to `UserAddressLine1` and `UserAddressLine2` instead. This can be taken up in a separate PR.

Reply:
So this naming was given by sdk, so did the same, will ask them and modify them in next pr